src/k4a_ros_device.cpp

```diff
@@ -32,477 +32,465 @@ using namespace std;
 using namespace visualization_msgs;
 #endif
 
-K4AROSDevice::K4AROSDevice(const NodeHandle &n, const NodeHandle &p) : k4a_device_(nullptr),
-                                                                       k4a_playback_handle_(nullptr),
+K4AROSDevice::K4AROSDevice(const NodeHandle& n, const NodeHandle& p)
+  : k4a_device_(nullptr),
+    k4a_playback_handle_(nullptr),
+// clang-format off
 #if defined(K4A_BODY_TRACKING)
-                                                                       k4abt_tracker_(nullptr),
+    k4abt_tracker_(nullptr),
 #endif
-                                                                       node_(n),
-                                                                       private_node_(p),
-                                                                       image_transport_(n),
-                                                                       last_capture_time_usec_(0),
-                                                                       last_imu_time_usec_(0),
-                                                                       imu_stream_end_of_file_(false)
+    // clang-format on
```

Comment:
Potentially mismatched whitespace for ` // clang-format on`

Reply:
Clang format makes its own rules, can't fix this one unfortunately